SpatialGDK/Source/SpatialGDK/Public/Utils/RPCContainer.h

```diff
@@ -35,7 +38,8 @@ class FRPCContainer
 	using RPCContainerType = TMap<ESchemaComponentType, FRPCMap>;
 
 	void ProcessRPCs(const FProcessRPCDelegate& FunctionToApply, FArrayOfParams& RPCList);
-	static bool ApplyFunction(const FProcessRPCDelegate& FunctionToApply, const FPendingRPCParams& Params);
+	static bool ApplyFunction(const FProcessRPCDelegate& FunctionToApply, FPendingRPCParams& Params);
 
 	RPCContainerType QueuedRPCs;
+	static const int32 SECONDS_TO_DROP_RPC = 4;
```

Comment:
Could you make this a float and put it in the `SpatialGDKSettings`?

Reply:
I agree. I also feel the default should be a lot shorter.

Reply:
What value you think suits best @m-samiec ?

Reply:
I'll put 3 seconds for now

Reply:
My opinion would be 1 second. I think if we have logs for when this is occurring, we can run a full playtest and evaluate the logs

Reply:
Changed to 1 second, will add logs.

Reply:
Moved to settings, added logs.